config/_ytt_lib/github.com/cloudfoundry/metric-proxy/config/values.yml

```diff
@@ -0,0 +1,18 @@
+#@data/values
+---
+system_domain: ""
+system_namespace: ""
+
+images:
+  metric_proxy: "oratos/metric-proxy"
```

Comment:
@kkburr Can you pin this image to the specific digest you'd like to include with this PR?

Reply:
Sure, we can do that if it's required. 👍